Silicon/IdavillePkg/Library/VTdLib/VTdLib.c

```diff
@@ -843,6 +838,7 @@ UpdateDmarHeader (
   // Check if CPU is in X2APIC mode
   //
   Dmar->Flags |= EFI_ACPI_DMAR_TABLE_FLAGS_X2APIC_OPT_OUT; // get from CPU's APIC mode (x2apiclib), check later
+  ZeroMem (&CpuidRegister, sizeof (CpuidRegister));
```

Comment:
@paaakash line 846 assigns initial value to cpuidregister. Coverity doesn't support assembly code hence it reports an issue. fyi